src/amo/sagas/reviews.js

```diff
@@ -376,6 +379,53 @@ function* deleteAddonReview({
   }
 }
 
+function* fetchLatestUserReview({
+  payload: { addonId, addonSlug, errorHandlerId, userId },
+}: FetchLatestUserReviewAction): Generator<any, any, any> {
```

Comment:
I saw @willdurand use the [`Saga` type](https://github.com/mozilla/addons-frontend/blob/master/src/core/types/sagas.js#L3) for this in a recent PR. Should we be doing that?

Reply:
How is that any more helpful? Also, I just learned that `Generator` is the equivalent of `Generator<any, any, any>` so maybe we should use that? In any case, our approach is that we do not use type safety for covering what a generator yields and receives.

Reply:
It is a meaningful alias (and it is shorter).

Reply:
It's not shorter when you count the import statement 😁 but that's fine, I can use it. I'll add a comment to the type definition to explain that we don't rely on type safety for covering what a saga yields and receives. Maybe that will help someone be less confused when they first see it.